### DISCUSSION TITLE
Do you know how to deploy a socket.io service on vercel?

### DISCUSSION THREAD

**Discussion by hezhongfeng** [vercel/next.js#70430] [Help] [answered]
### Summary

I know how to deploy a NestJS service，but dont know how to make  socket.io

**Comment by icyJoseph** [comment, marked as answer]
- https://github.com/orgs/vercel/discussions/422#discussioncomment-20840
- https://vercel.com/guides/do-vercel-serverless-functions-support-websocket-connections

**Comment by hezhongfeng** [comment]
Thanks.
